### PR TITLE
update Damion's lab and link

### PR DESCRIPTION
**Pull request by ddooley**

### CODE REVIEW

**Comment by cthoyt** [comment]
Would be more sustainable to point to your ORCID :)

**Comment by ddooley** [comment]
Good point Charles!  Done!

File: docs/Membership.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Current members of the OBO Foundry Operations Committee are (in alphabetical ord
  * Seth Carbon (T), Lawrence Berkeley National Laboratory, Berkeley, CA, USA
  * [Melanie Courtot](http://purl.org/net/mcourtot) (O, T), EMBL-EBI, Cambridge, UK 
  * Alexander Diehl (T), University at Buffalo, Buffalo, NY, USA
- * Damion Dooley (O), Hsiao Lab, Simon Fraser University, BC, Canada
+ * [Damion Dooley](https://orcid.org/my-orcid?orcid=0000-0002-8844-9165) (O), Centre for Infectious Disease Genomics and One Health, Simon Fraser University, BC, Canada
```
Comment (cthoyt) [inline docs/Membership.md:32]:
```suggestion
  * [Damion Dooley](https://orcid.org/0000-0002-8844-9165) (O), Centre for Infectious Disease Genomics and One Health, Simon Fraser University, BC, Canada
```

**Comment by nlharris** [comment]
This list needs some other adjustments too but I'll do a separate PR for those.

**Pull request merged**